zkevm-circuits/src/evm_circuit/execution/callop.rs

```diff
@@ -920,18 +920,14 @@ mod test {
             },
             |block, _tx| block.number(0xcafeu64),
         )
-        .unwrap()
-        .into();
-        assert_eq!(
-            run_test_circuit_geth_data::<Fr>(
-                block,
-                CircuitsParams {
-                    max_rws: 4500,
-                    ..Default::default()
-                }
-            ),
-            Ok(())
-        );
+        .unwrap();
+
+        CircuitTestBuilder::new_from_test_ctx(ctx)
+            .params(CircuitsParams {
+                max_rws: 4500,
+                ..Default::default()
+            })
+            .run();
```

Comment:
This is the last test that is not passing.

This is a really weird one. As basically, nothing has changed on it. And all the sudden, a constraint is getting triggered inthe `ConstraintBuilder::buiild_general_constraints`.

See: 
```rust
 // When all the keys in the current row and previous row are equal.
        self.condition(q.not_first_access.clone(), |cb| {
            cb.require_zero(
                "non-first access reads don't change value",
                q.is_read() * (q.rw_table.value.clone() - q.rw_table.value_prev.clone()),
            );
```